test/src/integration/basic.test.ts

```diff
@@ -63,6 +105,268 @@ describe("solo - 1 node", () => {
         expect(invoice).toEqual({ success: true });
     });
 
+    describe("Sending invalid parcels over the limits (general)", () => {
+        let parcelEncoded: any[];
+        beforeEach(async () => {
+            const nonce = await node.sdk.rpc.chain.getNonce(address);
+            const parcel = node.sdk.core
+                .createPaymentParcel({
+                    recipient,
+                    amount: 0
+                })
+                .sign({
+                    secret,
+                    fee: 10,
+                    nonce
+                });
+            parcelEncoded = parcel.toEncodeObject();
+        });
+
+        test.each(["0x01" + "0".repeat(64), "0x" + "f".repeat(128)])(
+            "nonce: %p",
+            async (nonce, done) => {
+                parcelEncoded[0] = nonce;
+                try {
+                    await node.sendSignedParcelWithRlpBytes(
+                        RLP.encode(parcelEncoded)
+                    );
+                    done.fail();
+                } catch (e) {
+                    expect(e).toEqual(ERROR.INVALID_RLP_TOO_BIG);
+                }
+                done();
+            }
+        );
+
+        test.each(["0x01" + "0".repeat(64), "0x" + "f".repeat(128)])(
+            "fee: %p",
+            async (fee, done) => {
+                parcelEncoded[1] = fee;
+                try {
+                    await node.sendSignedParcelWithRlpBytes(
+                        RLP.encode(parcelEncoded)
+                    );
+                    done.fail();
+                } catch (e) {
+                    expect(e).toEqual(ERROR.INVALID_RLP_TOO_BIG);
+                }
+                done();
+            }
+        );
+
+        test.each(["asdf", "tcc", "a"])(
+            "networkId: %p",
+            async (networkId, done) => {
+                parcelEncoded[2] = networkId;
+                try {
+                    await node.sendSignedParcelWithRlpBytes(
+                        RLP.encode(parcelEncoded)
+                    );
+                    done.fail();
+                } catch (e) {
+                    expect(e).toEqual(ERROR.INVALID_RLP_INVALID_LENGTH);
+                }
+                done();
+            }
+        );
+
```

Comment:
Could you add an INVALID_NETWORK_ID case? Any 2 characters string except for "tc" will do.

Reply:
Added.